overview.md

```diff
@@ -1,5 +1,7 @@
 ## Overview ##
 
+The Synopsys Software Integrity Group is now Black Duck®. If you are an existing Detect user, please manually re-install Black Duck Detect in order to receive future updates. If you are a new user, you may proceed with installing Black Duck Detect.
+
 The [Synopsys Detect](https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/62423113/Synopsys+Detect) plugin for Azure DevOps (formerly known as Hub Detect) supports Software Composition Analysis (SCA: open source software detection). It is architected to seamlessly integrate Synopsys Detect with Azure DevOps build and release pipelines. Synopsys Detect makes it easier to set up and scan code bases using a variety of languages and package managers across different application security techniques.  
```

Comment:
Don't know why the diff shows the above line as still referring to "Synopsys". Merged changes  few minutes ago... please ignore